src/index.js

```diff
@@ -3,8 +3,23 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { IntlProvider } from 'react-intl';
+import messages_es from './i18n/es.json';
+import messages_en from './i18n/en.json';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const messages = {
+  es: messages_es,
+  en: messages_en,
+};
+
+const language = 'es'; //TODO: this hardcoded data will depend by the app language
```

Comment:
Why don't we make it browser dependable for now? :) What do you think?

Reply:
Yes, I'm agree with you. I will apply this.